projects/packages/forms/src/contact-form/css/grunion.scss

```diff
@@ -854,7 +854,7 @@ on production builds, the attributes are being reordered, causing side-effects
 	outline: none;
 }
 
-.contact-form .is-style-animated .grunion-field-wrap:not(.no-label) > input,
+.contact-form .is-style-animated .grunion-field-wrap:not(.no-label) > input:not([type="checkbox"]):not([type="radio"]):not([type="range"]),
```

Comment:
In this case, we're specifically changing padding for checkbox/radio/range fields. I assume that's intended and we should test those directly.